db/schema.rb

```diff
@@ -13,6 +13,7 @@
 ActiveRecord::Schema[7.0].define(version: 2023_06_17_075921) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "timescaledb"
```

Comment:
why do we need this? @kiliczsh

Reply:
I have integrated with Railway they are using timescale db
will it create a performance issue?

Reply:
Removed timescaledb extension